cli/README.md

```diff
@@ -0,0 +1,176 @@
+<div align="center">
+    <img
+    width="100%"
+    src="https://github.com/roboflow/inference/blob/main/banner.png?raw=true"
+    />
+
+<br>
+
+[notebooks](https://github.com/roboflow/notebooks) | [inference](https://github.com/roboflow/inference) | [autodistill](https://github.com/autodistill/autodistill) | [collect](https://github.com/roboflow/roboflow-collect)
+
+<br>
+
+[![version](https://badge.fury.io/py/roboflow.svg)](https://badge.fury.io/py/inference-cli)
+[![downloads](https://img.shields.io/pypi/dm/inference-cli)](https://pypistats.org/packages/inference-cli)
+[![license](https://img.shields.io/pypi/l/inference-cli)](https://github.com/roboflow/inference/blob/main/LICENSE)
+[![python-version](https://img.shields.io/pypi/pyversions/inference-cli)](https://badge.fury.io/py/inference-cli)
+
+</div>
+
+# Roboflow Inference CLI
+
+[Roboflow](https://roboflow.com) Inference is an opinionated tool for running inference on state-of-the-art computer vision models. With no prior
+knowledge of machine learning or device-specific deployment, you can deploy a computer vision model to a range of devices and environments. Inference supports object detection, classification, and instance segmentation models, and running foundation models (CLIP and SAM).
+
+## 🎥 Inference in action
+
+Check out Inference running on a video of a football game:
+
+https://github.com/roboflow/inference/assets/37276661/121ab5f4-5970-4e78-8052-4b40f2eec173
+
+## 👩‍🏫 Examples
+
+The [`/examples` directory](https://github.com/roboflow/inference/blob/main/examples/) contains example code for working with and extending `inference`, including HTTP and UDP client code and an insights dashboard, along with community examples (PRs welcome)!
+
+### `inference serve`
+
+`inference serve` is the main command for starting a local inference server. It takes a port number and will only start the docker container if there is not already a container running on that port.
+
+```bash
+inference serve --port 9001
+```
+
+### `inference infer`
+
+`inference infer` is the main command for running inference on a single image. It takes a path to an image, a Roboflow project name, model version, and API key, and will return a JSON object with the model's predictions. You can also specify a host to run inference on our hosted inference server.
+
+#### Local image
+
+```bash
+inference infer --image ./image.jpg --project_id my-project --model-version 1 --api-key my-api-key
+```
+
+#### Hosted image
+
+```bash
+inference infer --image https://[your-hosted-image-url] --project_id my-project --model-version 1 --api-key my-api-key
+```
+
+#### Hosted inference
+
+```bash
+inference infer --image ./image.jpg --project_id my-project --model-version 1 --api-key my-api-key --host https://infer.roboflow.com
```

Comment:
I thought they were all the same, but I tested and `infer` doesn't work. Paul mentioned it might be set up to only work with core models. detect on the other hand is setup to work with all model types.. I'll switch to `.detect`.